dts/arm/st/f7/stm32f7.dtsi

```diff
@@ -634,14 +634,34 @@
 
 		adc1: adc@40012000 {
 			compatible = "st,stm32-adc";
-			reg = <0x40012000 0x400>;
+			reg = <0x40012000 0x50>;
```

Comment:
For some reason the length is always set to `0x400`. Maybe @erwango can knows what's the reason behind this?

Reply:
the per-ADC regspace is only 0x50, but there are four common registers related to all three of the ADCs. luckily the adc_stm32 driver does not access this common configuration space, which makes my hack work.



![ADC_regs](https://user-images.githubusercontent.com/151651/106269317-a61e0380-622c-11eb-9871-a0e182c03136.png)

Reply:
@martinjaeger The reg space per device instance is typically 0x400, so copy/paste after copy/paste it ended up here and since this value is not used in the code, no one saw this was not correct in this case.

Reply:
Ok, thanks. So it's good to use the correct space of `0x050` here.